covid_xprize/oxford_data/tests/test_oxford_data.py

```diff
@@ -69,6 +71,30 @@ def test_create_prediction_initial_context_and_action_vectors(self):
         self.assertEqual(last_day_of_training_cases_data, prediction_context[-1])
         gc.collect()
 
+    def test_convert_to_new_cases(self):
+        df = oxford_data.prepare_cases_dataframe(DATA_PATH)
```

Comment:
+1 for the unit test 👍 . A docstring explaining what it does would make it even better. I had to read the code to see it was converting smooth cases per 100k back to the original new cases.